src/app/games/lasca/LascaControlAndDomination.ts

```diff
@@ -0,0 +1,37 @@
+import { BoardValue } from 'src/app/jscaip/BoardValue';
+import { MGPNode } from 'src/app/jscaip/MGPNode';
+import { Player } from 'src/app/jscaip/Player';
+import { GameStatus } from 'src/app/jscaip/Rules';
+import { LascaControlMinimax } from './LascaControlMinimax';
+import { LascaMove } from './LascaMove';
+import { LascaRules } from './LascaRules';
+import { LascaSpace, LascaState } from './LascaState';
+
+class LascaNode extends MGPNode<LascaRules, LascaMove, LascaState> {}
+
+export class LascaControlAndDominationMinimax extends LascaControlMinimax {
+
+    public getBoardValue(node: LascaNode): BoardValue {
+        const gameStatus: GameStatus = this.ruler.getGameStatus(node);
+        if (gameStatus.isEndGame) {
+            return new BoardValue(gameStatus.toBoardValue());
+        }
+        const controlValue: number = super.getBoardValue(node).value * 12;
+        let dominatingPiecesCount: number = 0;
+        for (let y: number = 0; y < LascaState.SIZE; y++) {
+            for (let x: number = 0; x < LascaState.SIZE; x++) {
+                const space: LascaSpace = node.gameState.getPieceAtXY(x, y);
+                if (space.getStackSize() > 0) {
+                    const stackSize: number = space.getStackSize();
+                    let pieceIndex: number = 0;
+                    const commander: Player = space.getCommander().player;
+                    while (pieceIndex < stackSize && space.get(pieceIndex).player === commander) {
```

Comment:
boucle for is moar readable nein ?

Reply:
Les for vide c'est un peu crade, et du coup même un peu trop "monoline", içi au moins l'incrémentation est sur une autre ligne